tide/master/PixelStreamWindowManager.h

```diff
@@ -100,7 +100,7 @@ class PixelStreamWindowManager : public QObject
      * @param webbrowser create a webbrowser window.
      */
     void openWindow( const QString& uri, const QPointF& pos, const QSize& size,
-                     bool webbrowser = false );
+                     bool webbrowser = false, bool whiteboard = false );
```

Comment:
that is starting to make too many booleans with impossible combinations (it can't be a webbrowser AND a whiteboard at the same time). Replace with an enum and also adjust the function documentation.